unison-cli/src/Unison/CommandLine/Completion.hs

```diff
@@ -294,7 +296,16 @@ prefixCompletePatch ::
   String ->
   Path.Absolute -> -- Current path
   Sqlite.Transaction [Line.Completion]
-prefixCompletePatch = completeWithinNamespace (NESet.singleton PatchCompletion)
+prefixCompletePatch = completeWithinNamespace (NESet.singleton UpgradeCompletion)
+
+-- | Completes a uograde argument by prefix-matching against the query.
+prefixCompleteUpgrade ::
+  String ->
+  Path.Absolute -> -- Current path
+  Sqlite.Transaction [Line.Completion]
+prefixCompleteUpgrade = completeWithinNamespace (NESet.singleton PatchCompletion)
```

Comment:
It looks like the arguments for Upgrade and Patch got swapped here?

And then what happenes when `completeWithinNamespace` gets called with `UpgradeCompletion`? I haven't found the spot yet 😅